typings/csstype/index.d.ts

```diff
@@ -1,3 +1,4 @@
+import type * as CSS from 'csstype' // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
What is this import for?

Reply:
There was an error without it, I'll have to look it up at my computer tomorrow. It's also how module augmentation is done in the csstype docs (the third point here): https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors

Reply:
Gotcha. Can you add a link to the docs you shared here?

Reply:
`vue-tsc` gives the following error when we don't have the import:

```
src/components/VAudioTrack/layouts/VBoxLayout.vue(2,18): error TS2322: Type '{ width: string | undefined; }' is not assignable to type 'StyleValue | undefined'.
  Object literal may only specify known properties, and 'width' does not exist in type 'CSSProperties | StyleValue[]'.
src/components/VSkeleton/VAudioTrackSkeleton.vue(16,23): error TS2322: Type '{ width: string; }' is not assignable to type 'StyleValue | undefined'.
  Object literal may only specify known properties, and 'width' does not exist in type 'CSSProperties | StyleValue[]'.
src/components/VAudioTrack/VWaveform.vue(94,17): error TS2322: Type '{ width: string; left: string; }' is not assignable to type 'StyleValue | undefined'.
  Object literal may only specify known properties, and 'width' does not exist in type 'CSSProperties | StyleValue[]'.
```
I guess without the import it considers that `CSSProperties` type only has the properties that are in the `typings/csstype/index.d.ts` instead of augmenting it...